### PR TITLE
extundelete: add page

### PR DESCRIPTION
**Pull request by sawshep**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/extundelete.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# extundelete
+
+> Restore files from an unmounted ext3 or ext4 partition.
```
Comment (navarroaxel) [inline pages/linux/extundelete.md:3]: Maybe this is more clear:
```suggestion
> Recover deleted files from ext2, ext3 or ext4 partitions by parsing the journal.
```

File: pages/linux/extundelete.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> See also `date` for Unix time information and `umount` for unmounting partitions.
+> More information: <http://extundelete.sourceforge.net/>.
```
Comment (navarroaxel) [inline pages/linux/extundelete.md:5]:
```suggestion
> More information: <http://extundelete.sourceforge.net>.
```

File: pages/linux/extundelete.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Restore all deleted files inside partion 1 on device b (`sdb1`):
+
+`sudo extundelete {{/dev/sdb1}} --restore-all`
```
Comment (navarroaxel) [inline pages/linux/extundelete.md:9]:
```suggestion
- Restore all deleted files inside partition N on device X:

`sudo extundelete {{/dev/sdXN}} --restore-all`
```

maybe we can remove `N on device X`, I think only advanced users use this app.

File: pages/linux/extundelete.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- Restore a file from a path relative to root (Do not start the path with `/`):
+
+`sudo extundelete {{/dev/sdb1}} --restore-file {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/linux/extundelete.md:13]:
```suggestion
`extundelete {{/dev/sdXN}} --restore-file {{path/to/file}}`
```

File: pages/linux/extundelete.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- Restore a directory from a path relative to root (Do not start the path with `/`):
+
+`sudo extundelete {{/dev/sdb1}} --restore-directory {{path/to/directory}}`
```
Comment (navarroaxel) [inline pages/linux/extundelete.md:17]:
```suggestion
`extundelete {{/dev/sdXN}} --restore-directory {{path/to/directory}}`
```

File: pages/linux/extundelete.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Restore all files deleted after January 1st, 2020 (in Unix time):
+
+`sudo extundelete {{/dev/sdb1}} --restore-all --after {{1577840400}}`
```
Comment (navarroaxel) [inline pages/linux/extundelete.md:21]:
```suggestion
`extundelete {{/dev/sdXN}} --restore-all --after {{1577840400}}`
```

Based on the `date` page, looks like we don't template the Unix time variable,

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @sawshep, thanks for your contribution! I made a few comments.

File: pages/linux/extundelete.md (hunk 1 above)
Comment (sawshep) [reply pages/linux/extundelete.md:3]: I will add the rest of the suggestion, but this utility does not work with ext2 filesystems as they lack a journal.
```suggestion
> Recover deleted files from ext3 or ext4 partitions by parsing the journal.
```

Comment (navarroaxel) [reply pages/linux/extundelete.md]: 😮 so, Arch Linux has a bug in their package description https://www.archlinux.org/packages/community/x86_64/extundelete/

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @sawshep!

**Pull request merged**